addons/mail/static/src/core/messaging_service.js

```diff
@@ -716,6 +716,7 @@ export const messagingService = {
         "notification",
         "presence",
         "mail.attachment",
+        "mail.chat_window",
```

Comment:
chat window should be defined in web folder

Reply:
done! (removed)